.github/workflows/sbt-matrix.yml

```diff
@@ -22,16 +42,29 @@ jobs:
       matrix:
         java: ${{ fromJson(inputs.java) }}
         scala: ${{ fromJson(inputs.scala) }}
+        include: ${{ fromJson(inputs.include) }}
+        exclude: ${{ fromJson(inputs.exclude) }}
```

Comment:
what's the use case for this?

Reply:
It's just to support an existed GA matrix behavior. 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-additional-values-in-combinations
It can be useful in the future when we will want to have a more special matrix strategy than the cartesian product of two array with Java and Scala versions.